src/basic/string-table.c

```diff
@@ -8,7 +8,7 @@ ssize_t string_table_lookup(const char * const *table, size_t len, const char *k
                 return -EINVAL;
 
         for (size_t i = 0; i < len; ++i)
-                if (streq_ptr(table[i], key))
+                if (strcasecmp_ptr(table[i], key))
```

Comment:
This would change the parsing for _all_ tables, which seems too much. Just drop this commit, please. @yuwata if you want this upper/lower case change, please implement it yourself with backward compatibility, as it's unrelated to this or the other PR

Reply:
removed commit and changes all compression values to upper case, HTTP header values are case insensitive, so there should be no difference.